docs/usage.md

```diff
@@ -18,7 +16,7 @@ ds = xr.tutorial.open_dataset('air_temperature')
 ds.to_netcdf('air.nc')
 ```
 
-We can open a virtual representation of this file using {py:func}`open_virtual_dataset <virtualizarr.open_virtual_dataset>`.
```

Comment:
If we're going to break all API links we at least need an issue and a plan for putting them back in.

Reply:
See https://virtualizarr--615.org.readthedocs.build/en/615/usage/#opening-files-as-virtual-datasets for an example of how the API links work with mkdocs.

Reply:
I added all remaining links in [`257d72a` (#615)](https://github.com/zarr-developers/VirtualiZarr/pull/615/commits/257d72a4d33a7325291cdd00dc1bfa9197c3d37e)